test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionDetailsTests.cs

```diff
@@ -148,6 +151,7 @@ public void ConnectionDetailsOptionsShouldBeDefinedInConnectionProviderOptions()
             details.MaxPoolSize = expectedForInt + index++;
             details.MinPoolSize = expectedForInt + index++;
             details.PacketSize = expectedForInt + index++;
+            details.ColumnEncryptionSetting = expectedForStrings + index++;
```

Comment:
please add a test specifically for this new setting

Reply:
Done. Tests added to ConnectionServiceTests.cs